packages/donor/src/screens/authentication/AuthLoadingScreen.tsx

```diff
@@ -1,7 +1,7 @@
 import styles from "./AuthLoadingScreen.module.scss";
 import { Player } from "@lottiefiles/react-lottie-player";
 
-import startachLogo from "../../assets/mobile/startach_logo.svg";
+import startachLogo from "../../assets/mobile/startach_logo.png";
```

Comment:
SVG is better than PNG as it is a vector file and has no resolution issues.

Can you export the new logo as SVG?